functions.txt

```diff
@@ -8,6 +8,9 @@ if (0) {
   define('PHP_SAPI', php_sapi_name());//"Defined in source code"
 }
 
+require_once __DIR__ . '/functions_uberh3.txt';
+require_once __DIR__ . '/functions_spl.txt';
```

Comment:
Maybe, functions_exceptions.txt ?

Reply:
Then we could move Throwable and friends there too.
But my initial thought: since these "extra" exceptions are a part of SPL, we could name the file as such.
Although we don't have any other SPL classes (yet?).
https://www.php.net/manual/ru/spl.exceptions.php